charts/placement-policy-scheduler-plugins/templates/rbac.yaml

```diff
@@ -1,8 +1,24 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: pp-plugins-scheduler
+  name: aggregated-pp-plugins-scheduler-reader
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-view: "true"
+    rbac.authorization.k8s.io/aggregate-to-edit: "true"
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
```

Comment:
`rbac.authorization.k8s.io/aggregate-to-edit` and `rbac.authorization.k8s.io/aggregate-to-admin` are only for admin and edit roles. This is reader role, so we shouldn't have it here.